tokenizers/src/tokenizer/mod.rs

```diff
@@ -619,16 +619,6 @@ where
     ///
     /// Fails if `stride` is too high relative to `max_length` and `post_processor.added_tokens()`
     pub fn with_truncation(&mut self, trunc: Option<TruncationParams>) -> Result<&mut Self> {
-        if let Some(trunc_params) = &trunc {
```

Comment:
I don't feel great about modifying this sanitation. 
It seems to me that one should be aware of the added tokens as it's the standard way to tokenizer defined by the tokenizer's creator. So preventing chunking below those is kind of important.

If someone REALLY wants super low chunking and wants to ignore the added tokens. It seems specific enough that simply changing the post_processor for None would be much simpler at this points.

So we can keep this footgun check alive, and power users can still modify the tokenizers' behavior.